packages/bot-skeleton/src/scratch/dbot.js

```diff
@@ -94,7 +93,9 @@ class DBot {
                 window.dispatchEvent(new Event('resize'));
                 window.addEventListener('dragover', DBot.handleDragOver);
                 window.addEventListener('drop', e => DBot.handleDropOver(e, handleFileChange));
-                api_base.init();
+                if (api_call) {
+                    api_base.init();
+                }
```

Comment:
i have reverted this change ... actually i made this change because on every initialisation of bot we used to make a web socket connection so when i was trying to reinitialise the bot it used to make a new websocket connection to avoid that i had made a check but now that check is not required